src/Endatix.Api/Endpoints/FormDefinitions/GetActive.cs

```diff
@@ -4,13 +4,14 @@
 using Endatix.Api.Infrastructure;
 using Endatix.Core.Entities;
 using Endatix.Core.UseCases.FormDefinitions.GetActive;
+using Errors = Microsoft.AspNetCore.Mvc;
 
 namespace Endatix.Api.Endpoints.FormDefinitions;
 
 /// <summary>
 /// Endpoint for getting the active form definition.
 /// </summary>
-public class GetActive(IMediator _mediator) : Endpoint<GetActiveFormDefinitionRequest, Results<Ok<FormDefinitionModel>, BadRequest, NotFound>>
+public class GetActive(IMediator mediator) : Endpoint<GetActiveFormDefinitionRequest, Results<Ok<FormDefinitionModel>, BadRequest, NotFound<Errors.ProblemDetails>>>
```

Comment:
I believe such change makes sense in every endpoint that can return NotFound (probably for all the unsuccessful paths also?).
If this is the case, they I suggest we have such change for all the endpoints. Not necessarily as a part of this PR. I think the best place will be https://github.com/endatix/endatix/pull/136.

Reply:
Yes, the PR will address the issue. I have tested that this will not happen once #136 is merged. However, will need to investigate adding support for ProblemDetails in all errors as it makes the API control better. One way or another will unify things for the #136 and decide what to do with ProblemDetails support as it's good to have it if it's not too hard for devs to add it to every error response (should be the only option or none) - https://fast-endpoints.com/docs/configuration-settings#rfc7807-rfc9457-compatible-problem-details

Reply:
OK, seems reasonable. Let's finish with #136 and evaluate the options after.